src/wasm/wasm-binary.cpp

```diff
@@ -1511,23 +1524,27 @@ void WasmBinaryBuilder::readTableElements() {
 
 void WasmBinaryBuilder::readNames() {
   if (debug) std::cerr << "== readNames" << std::endl;
+  auto name_type = getU32LEB();
+  auto subsection_size = getU32LEB();
+  WASM_UNUSED(subsection_size);
```

Comment:
Just the latter, currently, but I'm looking at combining them when it makes sense to do so (I'm currently emitting `tee_local` at way too late a stage, by looking at the opcode stream for the right combinations, for example).